austation/code/modules/food_and_drinks/food/snacks_bread.dm

```diff
@@ -1,3 +1,56 @@
+/obj/item/reagent_containers/food/snacks/synthetic_cake
+	name = "Cake Foods Holder Obj"
+	desc = "If you can see this description the code for the cake printer fucked up."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = ""
+	bitesize = 3
+	list_reagents = list(/datum/reagent/consumable/nutriment = 4)
+	tastes = list("sweetness" = 2,"cake" = 5)
+	foodtype = GRAIN | DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/synthetic_cake/toxic
+	name = "Toxic Cake Foods Holder Obj"
+	desc = "If you can see this description the code for the cake printer fucked up."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = ""
+	bitesize = 3
+	list_reagents = list(/datum/reagent/toxin/chloralhydrate = 10)
+	tastes = list("sweetness" = 2,"cake" = 5)
+	foodtype = GRAIN | DAIRY | SUGAR
+
+/obj/item/reagent_containers/food/snacks/synthetic_cake/Initialize(mapload, obj/item/caked)
```

Comment:
aaaaa, don't run initialize for this, make it a separate proc xD

Reply:
no